src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -191,38 +191,37 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
       );
     }
 
-    if (this._configuration.enableUnifiedPlanForChromiumBasedBrowsers) {
-      if (browserBehavior.hasChromiumWebRTC()) {
-        this.logger.info('WebRTC unified plan for Chromium-based browsers is enabled');
-      } else {
-        this.logger.info(`WebRTC unified plan is required for ${browserBehavior.name()}`);
+    if (browserBehavior.requiresUnifiedPlan()) {
```

Comment:
This is kinda unnecessary. We should just need to check for Chrome here.

Reply:
sgtm